handlers/authentication.go

```diff
@@ -249,6 +249,48 @@ func performOpenIdAuthentication(w http.ResponseWriter, r *http.Request) bool {
 			RespondWithDetailedError(w, http.StatusForbidden, "the OpenID token was rejected", err.Error())
 			return true
 		}
+
+		if len(conf.Auth.OpenId.AllowedDomains) > 0 {
+			log.Debugf("Allowed Domains: %v", conf.Auth.OpenId.AllowedDomains)
+			tokenClaims := openIdParams.ParsedIdToken.Claims.(jwt.MapClaims)
+			var hostedDomain string
+			if v, ok := tokenClaims["hd"]; ok {
+				hostedDomain = v.(string)
+			} else {
+				//domains like gmail.com don't have the hosted domain (hd) on claims
+				//fields, so we try to get the domain on email claim
+				log.Debugf("Host Domain not found, trying to discover the host domain over e-mail")
```

Comment:
I'm thinking on user who needs to troubleshooting a issue with allowed domains.
If we remove this debug lines, we create a "hole" on informations of login and the users cannot see cleary what are happen.

Trace logs are too verbose, I don't think it will help on simple troubleshooting scenario.

But you have hore experience than me. If you should necessary remove this debug lines, we remove :)

Reply:
Pesonally, I would remove all logs, because it's more or less feasible to troubleshoot given the error message that is returned. 

So, if you choose to keep them, my suggestion is to use `log.Tracef`. This is because some people like to collect debug-level logs even in production. So, this is going to generate excesive logs for these people.